server.py

```diff
@@ -0,0 +1,19 @@
+from win32event import CreateMutex
+from win32api import CloseHandle, GetLastError
+from winerror import ERROR_ALREADY_EXISTS
+
+
+class SingleInstance:
+    """ Limits application to single instance """
+
+    def __init__(self):
+        self.mutexname = "testmutex_{D0E858DF-985E-4907-B7FB-8D732C3FC3B9}"
+        self.mutex = CreateMutex(None, False, self.mutexname)
+        self.lasterror = GetLastError()
+
+    def aleradyrunning(self):
```

Comment:
I realize that this comes (without attribution) from http://code.activestate.com/recipes/578966-creating-a-single-instance-application/ , but there is a typo. Should be `aleradyrunning`